### ISSUE TITLE
Use HtmlWebSocketChannel for Web-Channel

### ISSUE THREAD

**Issue by stud0r** [blackhorse-one/stomp_dart_client#5]
The current solution is not working for flutter_web.

Would it be possible to provide the channel or the implementation to switch it to HtmlWebSocketChannel?

**Comment by KammererTob** [comment]
I'll see what i can do.

**Comment by KammererTob** [comment]
@stud0r 
I created a first implementation on another branch. Since i currently have no setup to manually test this for web: Would you be able to do this?
Just import this repository directly into your project by referencing this new feature branch:

```
stomp_dart_client:
    git:
      url: git://github.com:blackhorse-one/stomp_dart_client.git
      ref: web-support
```

**Comment by stud0r** [comment]
Hi @KammererTob 
Great Work! I was able to successfully test it on the new branch.

**Comment by KammererTob** [comment]
@stud0r 
Thanks for testing this. I will merge and write some unit-tests and then release a new version for this.
I will let you know once released.

**Comment by KammererTob** [comment]
Released with 0.3.0
